tests/IResearch/IResearchAqlAnalyzer-test.cpp

```diff
@@ -93,7 +93,9 @@ TEST_F(IResearchAqlAnalyzerTest, test_create_valid) {
                                                VPackParser::fromJson("{\"queryString\": \"RETURN '1'\"}")->slice()),
                                              false);
     ASSERT_NE(nullptr, ptr);
+    ASSERT_FALSE(dynamic_cast<arangodb::iresearch::AqlAnalyzer*>(ptr.get())->isPreCalculated());
```

Comment:
Suggest to move that check inside assert_analyzer, so we could check the value right after the reset and not after analyzer was already exhausted